packages/itwinui-react/src/core/DatePicker/DatePicker.tsx

```diff
@@ -580,27 +632,45 @@ export const DatePicker = React.forwardRef((props, forwardedRef) => {
             </IconButton>
           )}
         </Box>
-        <Box className='iui-calendar-weekdays'>
+        <Box
+          as='div'
+          {...daysOfWeekProps}
+          className={cx('iui-calendar-weekdays', daysOfWeekProps?.className)}
+        >
           {shortDays.map((day, index) => (
-            <div key={day} title={longDays[index]}>
+            <div
+              key={day}
+              title={longDays[index]}
+              {...weekDayProps}
+              className={cx(className, weekDayProps?.className)}
+            >
               {day}
             </div>
           ))}
         </Box>
-        <div onKeyDown={handleCalendarKeyDown} role='listbox'>
+        <div
+          onKeyDown={handleCalendarKeyDown}
+          role='listbox'
+          {...listboxProps}
+          className={cx(className, listboxProps?.className)}
+        >
           {weeks.map((weekDays, weekIndex) => {
             return (
               <Box
+                as='div'
```

Comment:
What is the default of `<Box>` when no `as` prop is provided? Is it "div"?

Reply:
Yes, `<Box>` is `div` by default.